src/Categories/CategoryEntity.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace FurnitureStoreApi\Categories;
+header('Content-Type: application/json; charset=utf-8');
```

Comment:
This header should be set in your `categories.php`, ideally using a service (as mentioned before)